hack/update/update_all/update_all.go

```diff
@@ -50,7 +51,14 @@ func main() {
 		}
 
 		component := d.Name()
-		if component == "get_version" || component == "update_all" || component == "k8s-lib" || component == "amd_device_gpu_plugin_version" {
+		blackList := map[string]bool{
```

Comment:
Few years ago the entire world replaced terms like this. Why introduce them again?

The main issue with blacklist is that it does not describe well the purpose. It would be more helpful to call this "skip", so we can have code like:

```go
if skip[component] {
    continue
}